### DISCUSSION TITLE
matplotlib-charts example warns about nr of figures opened

### DISCUSSION THREAD

**Discussion by bigfooted** [Kitware/trame#349] [General]
The matplotlib example 05_charts/Matplotlib/00_matplotlib-charts.py will replot the figure when you resize the window. When you resize it for a couple of seconds, you will get the message:

`~/trame/examples/05_charts/Matplotlib/00_matplotlib-charts.py:45: RuntimeWarning: More than 20 figures have been opened. Figures created through the pyplot interface (`matplotlib.pyplot.figure`) are retained until explicitly closed and may consume too much memory. (To control this warning, see the rcParam `figure.max_open_warning`). Consider using `matplotlib.pyplot.close()`.
  fig, ax = plt.subplots(**figure_size())`

I just placed a
`plt.close('all')`
before the fig,ax=plt.subplots() line and I did not get this message anymore. I guess it could indeed be an issue when you let it run for a while?

**Comment by jourdain** [comment]
Do you mind doing a PR to update the example?
